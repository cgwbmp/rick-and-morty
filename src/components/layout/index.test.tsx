import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Layout from '.';

describe('<Layout />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Layout />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('DOM structure not changed', () => {
    const component = renderer.create(<Layout>Content</Layout>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
