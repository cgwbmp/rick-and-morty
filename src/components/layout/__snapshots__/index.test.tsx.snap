// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> DOM structure not changed 1`] = `
<div
  className="layout"
>
  <header
    className="header"
  >
    <h1
      className="title"
    >
      Welcome to The Rick and Morty Wiki!
    </h1>
  </header>
  Content
</div>
`;
